files/demo.hledger

```diff
@@ -0,0 +1,178 @@
+account Revenue                R
+account Expenses               X
+account Equity                 E
+
+
+# Let's declare some other accounts, so they will appear in the desired order
+# You can organize them in a tree structure using ':' to indicate a a parent-child relation account
+
+account Asset:Cash
+account Asset:Investments
+account Asset:Real Estate
+account Liabilities:Credit Card
+account Liabilities:Mortgage
+
+
+# Possibly you will not start from scratch. You will want to import your data from a previous software that you
+# want to adapt for the powerful hierarchical account system hledger provides. For that we will use alias so that
+# whenever hledger sees an old account, it will substitute for its new name
+
+alias jpm=Asset:Bank:JP Morgan Chase                     ; Format - alias OLD=NEW
+alias visa=Liabilities:Credit Card:Visa Platinum         ; Substitute 'visa' for 'Liabilities:Credit Card:Visa Platinum'
+
+
+# Now we will finally start adding transactions to our ledger. For that, you will need to understand the format. Don't
+# worry if it seems complicated. You will quickly be used to it, and hledger offers some great tools to make that
+# process easier with an interactive filling app with auto-complete and tools to automatically import data from your
+# bank or any other source. It may even automatically fill the data for you if it finds a similar older transaction!
+# If You you are curious, google hledger-iadd or hledger-autosync for some examples
+
+# A transaction is a financial event that has multiple posting showing where the money came and where it goes to.
+# The main property of a transaction is that it always need to balance, or in other words, it values should sum 0.
+# This property is called 'double-entry accounting'. It may seem hard to use, but hledger uses some ingenious approach
+# that makes it almost as simple as a regular 'single-entry-accounting' that you may be used to do.
+
+# Our first transaction will be the initial balances each account has one day before we effectively register transaction.
+# A good practice is to have one file for each year. Let's do it for 2018, so the date will be 31th, December 2017.
+
+2017/12/31 Initial Balance                               ; Format - date description. Date format - YYYY/MM/DD
+    Asset:Cash                                   300     ; <-- This is a posting: Format -   account  amount
+    ; Attention! You need to indent postings with at least 2 spaces.
+    ;            It is needed at least 2 spaces between the account and the amount
+    Asset:Bank:JP Morgan Chase                  4800.81  ; <-- After a ';' it is comment. It needs 2 spaces from the amount
+    Asset:Bank:Weels Fargo                      2400.23  ; <-- more posting. A transaction can have multiple postings!
+    Asset:Online Account:Apple Pay               120.34  ; <-- You don't need to declare accounts before using
+    Asset:Online Account:Paypal                   83,24  ; 
```

Comment:
Using a different decimal mark here. Should probably use period.

Reply:
Typo, that's the format in my country. Fixing